src/rgw/const.py

```diff
@@ -46,6 +46,39 @@
 SSL_CERT_CONFIGS = {"country" : "IN", "state" : "MH", "locality" : "Pune",
     "organization" : "Seagate Technology", "CN" : "seagate.com"}
 SSL_DNS_LIST = [u'*.seagate.com', u'localhost', u'*.localhost']
+SSL_CERT_PATH_KEY = 'cortx>common>security>ssl_certificate'
+RGW_ENDPOINT_KEY =  f'cortx>{COMPONENT_NAME}>service>endpoints'
```

Comment:
Avoid RGW in constants & variables in any new code, as name may change it later. 
We are driving in other PRs to replace RGW references with generic, lets not add it here again

Reply:
done. renamed variable/ method names from rgw - > svc